apps/condo/domains/miniapp/schema/fields/b2bAccessRightSet.js

```diff
@@ -0,0 +1,10 @@
+const { B2B_APP_SERVICE_USER_ACCESS_AVAILABLE_SCHEMAS } = require('../../utils/b2bAppServiceUserAccess/config')
```

Comment:
Use absolute import.

`./something` is fine
`../something` is ok, if part of the same module / block
`../../something`, probably not ok